src/daemons/GraphDaemon.cpp

```diff
@@ -33,7 +33,7 @@ using nebula::network::NetworkUtils;
 static void signalHandler(nebula::graph::GraphServer *graphServer, int sig);
 static Status setupSignalHandler(nebula::graph::GraphServer *graphServer);
 static void printHelp(const char *prog);
-#if defined(__x86_64__)
+#if defined(__x86_64__) && defined(ENABLE_BREAKPAD)
```

Comment:
Maybe we only need to check the `ENABLE_BREAKPAD` option, and the x86 arch could also be checked in `BreakpadConfig.cmake`

Reply:
It's not easy to get target arch.